backend/database/versions/05_67c4015bce73_add_custom_dataset_order.py

```diff
@@ -0,0 +1,28 @@
+"""add custom dataset order
+
+Revision ID: 05_67c4015bce73
+Revises: 04_2f30f3bcc9aa
+Create Date: 2024-01-25 11:42:46.126701
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "05_67c4015bce73"
+down_revision = "04_2f30f3bcc9aa"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        "CollectionVersion",
+        sa.Column("custom_dataset_order", sa.BOOLEAN(), nullable=True, server_default=sa.sql.expression.false()),
```

Comment:
I'm slightly confused. I thought we had agreed to instantiate this / treat this in the same fashion as the [boolean tombstone](https://github.com/chanzuckerberg/single-cell-data-portal/blob/317080ab15531c043f326574ea5c2c0bf409e031/backend/database/versions/03_98cf7564214f_add_tombstone_to_datasets.py)

Reply:
Per our conversations in Slack

Reply:
Also this column really should be given an assertive statement as a variable name, in keeping with conventions for boolean values (and yes I realize that `tombstone` should be `is_tombstoned` or something like that! 😭 ). Maybe `use_custom_dataset_order` or `has_custom_dataset_order`? Was this overruled somewhere in discussions? Can't remember if I suggested/mentioned this.

Reply:
Hey Dan! I updated the column as per your initial suggestion [here](https://github.com/chanzuckerberg/single-cell-data-portal/pull/6579#discussion_r1470536403) but can make another update: what is your preference between `use_custom_dataset_order` and `has_custom_dataset_order`?